MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/MvxRecyclerViewHolder.cs

```diff
@@ -136,9 +144,10 @@ private void OnItemViewOnLongClick(object sender, View.LongClickEventArgs args)
             this.ExecuteCommandOnItem(this.LongClick);
         }
 
-        public MvxRecyclerViewHolder(View itemView, IMvxAndroidBindingContext context)
+        public MvxRecyclerViewHolder(View itemView, int viewType, IMvxAndroidBindingContext context)
```

Comment:
Would it make more sense to switch the order of `viewType` and `context` and default `viewType to 0?